test/parallel/test-net-socket-connect-invalid-autoselectfamilyattempttimeout.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+
+const autoSelectFamilyAttemptTimeout = common.platformTimeout(-10);
+
+assert.throws(() => {
+  net.connect({
+    port: 8080,
+    autoSelectFamily: true,
+    autoSelectFamilyAttemptTimeout,
+  });
+}, { code: 'ERR_OUT_OF_RANGE' });
```

Comment:
`platformTimeout` serves no purpose here and might not be well-defined for negative numbers. I'd suggest hard-coding a negative integer and zero:

```suggestion
require('../common');
const assert = require('assert');
const net = require('net');

for (const autoSelectFamilyAttemptTimeout of [-10, 0]) {
  assert.throws(() => {
    net.connect({
      port: 8080,
      autoSelectFamily: true,
      autoSelectFamilyAttemptTimeout,
    });
  }, { code: 'ERR_OUT_OF_RANGE' });
}
```

Reply:
Applied your suggestion. Thank you for explanation and suggestion.